jplag/src/main/java/de/jplag/reportingV2/reportobject/model/JPlagReport.java

```diff
@@ -0,0 +1,21 @@
+package de.jplag.reportingV2.reportobject.model;
+
+import java.util.List;
+
+public class JPlagReport {
+    private final OverviewReport overviewReport;
+    private final List<ComparisonReport> comparisons;
+
+    public JPlagReport(OverviewReport overviewReport, List<ComparisonReport> comparisons) {
+        this.overviewReport = overviewReport;
+        this.comparisons = comparisons;
```

Comment:
you should copy lists. (if comparison results are mutable, you should also copy the elements of the list)